main.js

```diff
@@ -0,0 +1,33 @@
+class Person {
+    constructor(name, date, amount) {
+        this.transactions = [];
+        this.name = name;
+        this.date = date;
+        this.amount = amount;
+        const initial = `Initial: ${this.amount}`;
+        this.transactions = [this.transactions, initial];

```

Comment:
You missed `...`, but in this case will be better to use push, not need to reassign property every time